blog/dec-quiet-month-22.md

```diff
@@ -0,0 +1,41 @@
+---
+title: A Quiet Place (Dec'22)
+date: 2022-11-18T00:00:00.000Z
+authors:
+  name: plainekevin
+  url: 'https://github.com/PlaineKevin'
+  image_url: 'https://github.com/plainekevin.png?size=96'
+slug: a-quiet-place-22
+---
+
+The Electron project will pause for the month of December 2022, then return to full speed in January 2023.
+
+<div style={{width:"100%",height:"0",paddingBottom:"42%",position:"relative"}}>
+  <iframe src="https://giphy.com/embed/lbQQ1yeN4Z2wGm9q4d" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
+</div>
+<p><a href="https://giphy.com/gifs/marvelstudios-marvel-loki-studios-lbQQ1yeN4Z2wGm9q4d">via GIPHY</a></p>
+
+---
+
+## What will be the same in December
```

Comment:
I think an intro paragraph summarizing all the work we've done this year would be great. :)

Reply:
While I don't necessarily disagree, I think the point of this post should focus on letting everyone know we're not going to be as responsive this month. A post celebrating our wins would definitely be awesome but maybe shouldn't exist in the context of what this blog post is hoping to accomplish. Let me know if you disagree!